corehq/apps/app_manager/app_translations/parser.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import absolute_import
+
+import re
+from datetime import datetime
+from openpyxl import Workbook
+from tempfile import NamedTemporaryFile
+
+from corehq.apps.app_manager.app_translations.const import MODULES_AND_FORMS_SHEET_NAME
+from corehq.apps.dump_reload.const import DATETIME_FORMAT
+
+
+class TranslationsParser:
```

Comment:
I think this should be
```python
class TranslationsParser(object):
```

Reply:
hmm, i have started doing this recently. Is there an advantage by using object here?